src/server/zset_family.cc

```diff
@@ -1109,26 +1198,26 @@ void ZSetFamily::ZIncrBy(CmdArgList args, ConnectionContext* cntx) {
 
 void ZSetFamily::ZInterStore(CmdArgList args, ConnectionContext* cntx) {
   string_view dest_key = ArgS(args, 1);
-  OpResult<StoreArgs> store_args_res = ParseStoreArgs(args);
+  OpResult<UnionArgs> union_args_res = ParseUnionArgs(args, true);
```

Comment:
calling it UnionArgs and then use it in Inter method is a bit confusing. Why not `SetOpArgs` since they are shared by Set operations?

Reply:
Renamed `UnionArgs` to `SetOpArgs` as part of https://github.com/dragonflydb/dragonfly/pull/420/commits/e44d27b75530be3c97e787eca68957f7b1a3f088